server/src/main/java/io/druid/server/http/CoordinatorDynamicConfigsResource.java

```diff
@@ -58,11 +71,29 @@ public Response getDynamicConfigs()
 
   @POST
   @Consumes(MediaType.APPLICATION_JSON)
-  public Response setDynamicConfigs(final CoordinatorDynamicConfig dynamicConfig)
+  public Response setDynamicConfigs(final CoordinatorDynamicConfig dynamicConfig,
```

Comment:
if the audit info is not set it will throw exception, right now its mandatory to specify the audit info. 
do you think its worth making it optional or configurable ?